### ISSUE TITLE
FusedBatchNormV3 and AddV2 need custom implementations.

### ISSUE THREAD

**Issue by Jukyy** [tensorflow/tensorflow#35901]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 Pro, Version 1903
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): v2.1.0-rc2-17-ge5bf8de410 2.1.0
- Python version: 3.7
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: 10.2.89
- GPU model and memory: GeForce GTX 1050 2.00GiB

**Describe the current behavior**

I have created the simplest case of sequential neural network which only contains one Batch Normalization layer. Then I have tried to convert this model to Tensorflow Lite but then I got following message:

    Here is a list of operators for which you will need custom implementations: FusedBatchNormV3.

When I tried to set parameter `fused=False` I got following message:

    Here is a list of operators for which you will need custom implementations: AddV2.

**Describe the expected behavior**

I expect converted Tensorflow Lite model with optimized Batch Normalization weights.

**Code to reproduce the issue**

    import tensorflow as tf

    tflite_path = 'test.tflite'
    keras_path = 'test.h5'

    bn = tf.keras.layers.BatchNormalization(
        # fused=False,
        input_shape = (128,128,3)
    )

    # Create model
    k_model = tf.keras.models.Sequential()
    k_model.add(bn)
    k_model.save(keras_path)
    
    # Convert keras model to tflite
    converter = tf.lite.TFLiteConverter.from_keras_model(k_model)
    tflite_model = converter.convert()
    open(tflite_path, 'wb').write(tflite_model)

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by gargn** [comment]
This model appears to convert with `tf-nightly` (`pip install tf-nightly`) version `2.1.0.dev20200102`. We also recommend using `converter.experimental_new_converter = True`. However, that didn't seem necessary for me to get the model converting.

**Comment by Jukyy** [comment]
I have an update regarding this issue. 

I have tried installing `tf-nightly` version `2.1.0.dev20200102` on existing machine but the problem was still there.
When I tried to run the same script on a different machine (Windows 10 machine version 1903 with clean `Anaconda3 Python 3.7` installation) both `tf-nightly` and `tensorflow 2.1` were working correctly (exactly as I described in expected behavior).

**Comment by gargn** [comment]
Taking a closer look into the op, `FusedBatchNormV3` has been supported since June 2019. Given that this issue is only occurring in one installation of TensorFlow, it appears this is an installation related issue. Can you try reinstalling TensorFlow on the machine giving the error or provide additional information about your installation that might help with debugging?

**Comment by Jukyy** [comment]
I was able to find a core problem.
Last year I have built `Tensorflow 1.13` python package from source. During installation python had prompted me to use `--user` flag for installation.  By using `--user` flag TensorFlow was installed in `AppData/Roaming/Python` location. Because of that python interpreter was referencing some TensorFlow python scripts from `AppData/Roaming/Python` location despite me crating clean new virtual environment.
I had solved this issue by deleting TensorFlow from `AppData/Roaming/Python` location.
Thank you for your assistance.

**Comment by ymodak** [comment]
Closing this issue now. Thanks!

**Comment by gargn** [comment]
Closing because it is resolved as an installation issue.
